test/Tmdb/Tests/Repository/MovieRepositoryTest.php

```diff
@@ -30,7 +30,7 @@ public function shouldLoadMovie()
         $repository->load(self::MOVIE_ID);
         $this->assertLastRequestIsWithPathAndMethod('/3/movie/' . self::MOVIE_ID);
         $this->assertRequestHasQueryParameters([
-            'append_to_response' => 'alternative_titles,external_ids,changes,credits,images,keywords,lists,release_dates,reviews,similar,recommendations,translations,videos'
+            'append_to_response' => 'alternative_titles,external_ids,changes,credits,images,keywords,lists,release_dates,reviews,similar,recommendations,translations,videos,watch/providers'
```

Comment:
Any chance you could update the stored data ( flat json files ) that include this data and add some specific tests? :-)